package.json

```diff
@@ -177,7 +180,7 @@
     "sleep-promise": "8.0.1",
     "tempy": "0.2.1",
     "text-table": "0.2.0",
-    "through2": "3.0.0",
+    "through2": "^3.0.0",
```

Comment:
Let's not use the hat version here, can we specify the exact pinned version (`3.0.0`).

Reply:
removed